ros2/src/airsim_ros_pkgs/include/math_common.h

```diff
@@ -42,4 +42,13 @@ inline T angular_dist(T from, T to)
         d += 2. * M_PI;
     return d;
 }
+
+template <typename T>
+inline T constrain_euler_angle_to_360(const T angle)
+{
+    T angle_constrained = angle;
+    angle_constrained -= 360. * std::floor(angle_constrained * (1. / 360.));
+    return angle_constrained;
+}
+
 }
```

Comment:
I think this takes care of corners

template <typename T>
inline T constrain_euler_angle_to_360(const T angle) {
    T angle_constrained = std::fmod(angle, 360);
    if (angle_constrained < 0) {
        angle_constrained += 360;
    }
    return angle_constrained;
}

Reply:
Updated.